CHANGELOG.md

```diff
@@ -48,6 +50,66 @@
 
 Although every attempt has been made to make this information as accurate as possible, please know there may be things that are omitted, not fully developed yet, or updates since this publication that were not included in the information below. Only the most pressing or significant items have been listed. For the entire list of tickets and or specific information about any given item, please visit the list at [Hyperleder Indy's Jira](https://jira.hyperledger.org/). Once logged in, simply navigate to Projects > Indy.
 
+## 1.10.0
+### Release date: Oct 4th, 2019
+
+### Component Version Information
+| Components | Version Numbers |
+| --- | --- |
+| indy-plenum | 1.10.0 |
+| indy-node | 1.10.0 |
+| sovrin | 1.1.58 |
+
+### Additional Information:
+**There are possible OOM issues during 3+ hours of target load or large catch-ups at 8 GB RAM nodes pool so 32 GB is recommended.**
+**PBFT View Change was implemented but not enabled so old View Change is active now.**
+
+### Major Changes
+- PBFT View Change implementation
+- BLS fixes and improvements
+- Stability fixes
+
```

Comment:
Please mention ZMQ bump as well in Major Changes

Reply:
Done.